ros2_batch_job/__main__.py

```diff
@@ -426,9 +426,11 @@ def run(args, build_function, blacklisted_package_names=None):
                 info("'{0}' returned exit code '{1}'", fargs=(" ".join(vcs_custom_cmd), ret))
                 print()
 
-                # Attempt to rebase all the repositories to the __ci_default branch
-                info("Attempting to rebase all repositories to the '__ci_default' branch")
-                vcs_custom_cmd = vcs_cmd + ['custom', '.', '--git', '--args', 'rebase', '__ci_default']
+                # Attempt to merge all the repositories to the __ci_default branch.  This is to
+                # ensure that the changes on the branch still work when applied to the
+                # latest version of the default branch.
```

Comment:
@dirk-thomas You are absolutely right about that.  As far as the code goes, it doesn't matter; either way will produce the same result.  I'll fix up the comment so that it says the right thing.  Thanks.